client/my-sites/stats/post-trends/day.jsx

```diff
@@ -29,18 +31,32 @@ class PostTrendsDay extends PureComponent {
 	/* eslint-disable wpcalypso/jsx-classname-namespace */
 	buildTooltipData = () => {
 		const { label, postCount } = this.props;
-		const content = this.props.translate( '%(posts)d post', '%(posts)d posts', {
-			count: postCount,
-			args: {
-				posts: postCount,
-			},
-			comment: 'How many posts published on a certain date.',
-		} );
+		const date = moment( label );
+		const weekDay = date.format( 'dddd' );
+		const formattedDate = date.format( 'MMMM DD, YYYY' );
```

Comment:
Could we avoid using moment and use [`toLocaleDateString`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString) instead?

Our dependency on the moment library incurs a heavy bundle size penalty that we're working on alleviating.

Reply:
Sure! Just used `moment` because I saw it was being used on other components and thought it would be ok. I just implemented the suggested change.